arangod/VocBase/Methods/UpgradeTasks.cpp

```diff
@@ -545,24 +544,8 @@ bool UpgradeTasks::addDefaultUserOther(TRI_vocbase_t& vocbase,
   return true;
 }
 
-bool UpgradeTasks::persistLocalDocumentIds(TRI_vocbase_t& vocbase,
-                                           arangodb::velocypack::Slice const& slice) {
-  if (EngineSelectorFeature::engineName() != MMFilesEngine::EngineName) {
```

Comment:
I'd prefer to leave at least assertion for the future

Reply:
Mmmh, the entire method has been deleted. Where to put the assertion?

Reply:
:-)